test/runtests.jl

```diff
@@ -9,7 +9,7 @@ using Random
 
 w = CWorld()
 
-@inferred gen(DDNOut(:sp, :r), w, [3,2], [4,2], MersenneTwister(19))
+@gen(:sp, :r)(w, [3,2], [4,2], MersenneTwister(19))
```

Comment:
I think we can still use `@inferred` here. Does it fail the test?

Reply:
Yes, you're right, we should keep the ``@inferred`` here.  I just tested it and it passes.  Can you suggest the edit and I'll commit it? Thanks!

Reply:
Haha, I don't think I need to suggest it - there should be a way for you to just make the change, even on the github interface.

Reply:
In any case, I think it should be
```suggestion
@inferred @gen(:sp, :r)(w, [3,2], [4,2], MersenneTwister(19))
```

Reply:
Thanks! I can't seem to find the github button to suggest the change on my side...